spec/display_spec.rb

```diff
@@ -0,0 +1,56 @@
+describe Display do
+  describe '#view_menu' do
+    it 'shows the menu in nice format' do
+      expect(subject.view_menu(Menu::PIZZA)).to include("Marinara: £8.00")
+    end
+  end
+
+  describe '#view_order' do
+    let(:order) { [{ bufalina: 13 }, { diavola: 14 }] }
+
+    it 'displays the order in nice format' do
+      expect(subject.view_order(order)).to include("Diavola: £14.00")
+    end
+
+    it 'displays the total in nice format' do
+      expect(subject.view_order(order)).to include("Total: £27.00")
```

Comment:
these tests are good, I was having a lot of trouble trying to get rspec to test specific strings so this is v useful to see!